scripts/cli/bettererResultsToJson.ts

```diff
@@ -0,0 +1,42 @@
+import { betterer } from '@betterer/betterer';
+import { writeFile } from 'fs/promises';
+
+interface Issue {
+  message: string;
+  count: string;
+}
+
+type ResultMap = Record<string, Record<string, Issue[]>>;
+
+/**
+ *  Produces a JSON file for consumption directly in Grafana
+ */
+async function main() {
+  const results = await betterer.results();
+  const resultMap: ResultMap = {};
+
+  for (const suite of results.resultSummaries) {
+    resultMap[suite.name] = {};
+
+    // Aggregate issues for each file in the suite
+    for (const [file, details] of Object.entries(suite.details)) {
+      const fileIssues: Issue[] = [];
+      for (const issue of details) {
+        const issueExists = fileIssues.find((i) => i.message === issue.message)!!;
+        if (issueExists) {
+          continue;
+        }
+        fileIssues.push({
+          message: issue.message,
+          count: details.filter((i) => i.message === issue.message).length,
```

Comment:
I think the types are off here, getting `Parameter i implicitly has an any type.` and it's not being picked up by our lint rules since it's outside the public dir I assume.

Reply:
You're right, VSCode mentioned this. I find it weird that it doesn't get the type `BettererFileIssue` from betterer.